workflow/workflow_commands.go

```diff
@@ -1445,6 +1448,76 @@ func TraceWorkflow(c *cli.Context) error {
 	return nil
 }
 
+func UpdateWorkflow(c *cli.Context) error {
+	wid := c.String(common.FlagWorkflowID)
+	rid := c.String(common.FlagRunID)
+	name := c.String(common.FlagName)
+	uid := c.String(common.FlagUpdateID)
+	first_execution_run_id := c.String(common.FlagUpdateFirstExecutionRunID)
+	args, err := common.UnmarshalInputsFromCLI(c)
+	if err != nil {
+		return err
+	}
+	waitPolicy, ok := updateWaitPolicyMap[c.String(common.FlagUpdateWaitPolicy)]
+	if !ok {
+		return fmt.Errorf("must specify valid wait policy: %v", strings.Join(mapKeysToArray(updateWaitPolicyMap), ", "))
+	}
+	if (len(name) == 0 && len(uid) == 0) || (len(name) > 0 && len(uid) > 0) {
```

Comment:
You can simplify this to a XOR: `len(name) > 0 != len(uid) > 0`

Reply:
I think we're getting rid of this check, but just to correct myself, the condition I wrote above should be negated to XNOR because this condition evaluates to true iff we consider the arguments _invalid_, not valid.